### PR TITLE
feat: install gcal + gvideo by default for google signups

### PR DESCRIPTION
**Pull request by hbjORbj**
## What does this PR do?

- I am chunkifying the new onboarding flow (#17251) into smaller PRs
- This PR does NOT include any UI changes. It installs google calendar and google meets by default for google signups.

https://github.com/user-attachments/assets/6c510fe1-7cc0-4e8f-82d5-a209ace4d47f

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A - I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Go through a sign up process using Google sign-in. DM me if you need help with this.

### CODE REVIEW

File: packages/lib/server/repository/google.ts [hunk 1]
```diff
@@ -1,48 +0,0 @@
-import type { Prisma } from "@prisma/client";
-import type { Credentials } from "google-auth-library";
-
-import { CredentialRepository } from "./credential";
-import { SelectedCalendarRepository } from "./selectedCalendar";
-
-export class GoogleRepository {
```
Comment (hbjORbj) [inline packages/lib/server/repository/google.ts:7]: this is renamed to `GoogleService` and moved to `/service` folder. All methods here are copied into the new class.

File: packages/app-store/googlecalendar/lib/utils.ts [hunk 2]
```diff
@@ -1,29 +0,0 @@
-import type { calendar_v3 } from "googleapis";
-
-import logger from "@calcom/lib/logger";
-
-export async function getAllCalendars(
```
Comment (hbjORbj) [inline packages/app-store/googlecalendar/lib/utils.ts:5]: this is replaced by `GoogleService.getAllCalendars` now

File: packages/app-store/googlecalendar/lib/constants.ts [hunk 3]
```diff
@@ -1,7 +0,0 @@
-export const SCOPE_USERINFO_PROFILE = "https://www.googleapis.com/auth/userinfo.profile";
-export const SCOPE_CALENDAR_READONLY = "https://www.googleapis.com/auth/calendar.readonly";
-export const SCOPE_CALENDAR_EVENT = "https://www.googleapis.com/auth/calendar.events";
-
-export const REQUIRED_SCOPES = [SCOPE_CALENDAR_READONLY, SCOPE_CALENDAR_EVENT];
-
-export const SCOPES = [...REQUIRED_SCOPES, SCOPE_USERINFO_PROFILE];
```
Comment (hbjORbj) [inline packages/app-store/googlecalendar/lib/constants.ts:7]: all these values can be accessed by `packages/lib/constant.ts`

File: packages/lib/server/service/google.ts [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      } while (pageToken);
+
+      return allCalendars;
+    } catch (error) {
+      logger.error("Error fetching all Google Calendars", { error });
+      throw error;
+    }
+  }
+
+  static async getPrimaryCalendar(
```
Comment (hbjORbj) [inline packages/lib/server/service/google.ts:90]: I am intentionally not using `getAllCalendars` inside this method for optimization

**Review by zomars — APPROVED** [review]
LGTM

**Comment by zomars** [comment]
@hbjORbj  seems like this is causing OOM issues on Vercel. We might need to revert this

**Pull request merged**